Sources/SwiftFrame/main.swift

```diff
@@ -4,42 +4,36 @@ import Foundation
 import SwiftFrameCore
 import ArgumentParser
 
-extension URL: ExpressibleByArgument {
-
-    public init?(argument: String) {
-        self.init(fileURLWithPath: argument)
-    }
-
-}
-
-// Struct name serves as command name
 struct SwiftFrame: ParsableCommand {
 
     static let configuration = CommandConfiguration(
         commandName: "swiftframe",
         abstract: "CLI application for speedy screenshot framing",
-        version: "2.2.0",
+        version: "3.0.0",
         helpNames: .shortAndLong)
 
+    @Argument(help: "Read configuration values from the specified file", completion: .list(["config", "yml", "yaml"]))
```

Comment:
I think it makes more sense to have this as an argument, as it is a requirement rather than an option